refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/nfcconverter/NfcHelper.java

```diff
@@ -79,7 +79,7 @@ public static String[] nfcConverter(final String[] original) {
      * @see Normalizer#normalize(CharSequence, Normalizer.Form)
      */
     public static Map<String, String[]> nfcConverter(final Map<String, String[]> original) {
-        final HashMap<String, String[]> nfcConverted = new HashMap<>(original.size());
+        final Map<String, String[]> nfcConverted = new HashMap<>(original.size());
```

Comment:
I guess the refactorings were done due to `LooseCoupling` right? If so, we should get rid of all `LooseCoupling` suppressions if posslble.

Reply:
I think i replaced all except for beans, where it isn't sensibly possible in my opinion.